packages/editor/src/App.css

```diff
@@ -37,6 +41,7 @@
   --error-border: 1px solid var(--error-color);
   --seperator-border: var(--basic-border);
   --icon-seperator-line: 1px solid var(--N300);
+  --border-radio-checkbox: 1px solid var(--A75);
```

Comment:
I know not invented on this PR. But I recognized this now so can you improve this now or in the next PR?

Streamline the names of these variables, 'border' at the start or the end but not both.
--error-border
--border-radio-checkbox

Reply:
i would always put the css specific hint (what this variable is about) at the end.
E.g. radio-checkboy-border, radio-checkboy-background, warning-color, selected-text-color, etc.